Documentation/Running.rst

```diff
@@ -53,91 +55,91 @@ You can currently run these code checks on the command line:
 .. index:: Commands; composer ci:composer:normalize
 .. code-block:: bash
 
-    composer ci:composer:normalize
+    ./Build/Scripts/runTests.sh -s composer ci:composer:normalize
 
 Checks the composer.json.
 
 .. index:: Commands; composer ci:json:lint
 .. code-block:: bash
 
-    composer ci:json:lint
+    ./Build/Scripts/runTests.sh -s composer ci:json:lint
 
 Lints the JSON files.
 
 .. index:: Commands; composer ci:php
 .. code-block:: bash
 
-    composer ci:php
+ ./Build/Scripts/runTests.sh -s composer ci:php
```

Comment:
The indentation for code blocks here seems inconsistent to me: Here, it's 2 spaces, while above, it's 4 spaces.

This should be consistent in all changed lines.

The current recommendation is 4 spaces AFAIK (at least that's what the later core docs use).